src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1506,6 +1509,10 @@ private function registerMessengerConfiguration(array $config, ContainerBuilder
         $container->getDefinition('messenger.asynchronous.routing.sender_locator')->replaceArgument(1, $messageToSenderIdsMapping);
 
         foreach ($config['transports'] as $name => $transport) {
+            if (0 === strpos($transport['dsn'], 'amqp://') && !$container->hasDefinition('messenger.transport.amqp.factory')) {
+                throw new LogicException('The default AMQP transport is not available. Make sure you have installed and enabled the Serializer component. Try enable it or install it by running "composer require symfony/serializer".');
```

Comment:
We need the `serializer-pack` I think, just the Serializer wouldn't be enough (i.e. missing normalisers)

Reply:
The [serializer-pack](https://packagist.org/packages/symfony/serializer-pack) does not install any extra normalizer.
But I guess you meant in order for the [ObjectNormalizer denormalization to work efficiently](http://symfony.com/doc/current/components/serializer.html#recursive-denormalization-and-type-safety) based on typehints/docblocks, for which the PropertyInfo component and `phpdocumentor/reflection-docblock` are required.
So 👍 for mentioning the pack, but that's also probably an hint we already miss when just using the Serializer component.

Reply:
Changed to `-pack`.